webapp/packages/core-connections/src/DatabaseAuthDialog/SSHAuthForm.tsx

```diff
@@ -0,0 +1,85 @@
+/*
+ * CloudBeaver - Cloud Database Manager
+ * Copyright (C) 2020-2021 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0.
+ * you may not use this file except in compliance with the License.
+ */
+
+import { observer } from 'mobx-react-lite';
+import React from 'react';
+import styled, { css } from 'reshadow';
+
+import { FieldCheckbox, FormGroup, InputField, InputGroup } from '@cloudbeaver/core-blocks';
+import { useTranslate } from '@cloudbeaver/core-localization';
+import type { NetworkHandlerConfig } from '@cloudbeaver/core-sdk';
+
+import type { DBAuthDialogController } from './DBAuthDialogController';
+
+interface Props {
+  controller: DBAuthDialogController;
```

Comment:
pass `config` and `disaled =  controller.isAuthenticating` instead of controller